core/src/main/java/org/openconnectors/util/BasicCopyTopology.java

```diff
@@ -20,29 +20,57 @@
 package org.openconnectors.util;
 
 import org.openconnectors.config.Config;
-import org.openconnectors.connect.SourceConnector;
+import org.openconnectors.connect.PushSourceConnector;
+import org.openconnectors.connect.SinkConnector;
+import org.openconnectors.connect.PullSourceConnector;
+
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.function.Function;
 
 /**
  * Copy Topology template targeting unit testing
  */
-public class BasicCopyTopology<T> {
+public class BasicCopyTopology<I, O> {
```

Comment:
Can we call it something else than Topology?  May confuse people into thinking this is a storm/heron topology

Reply:
My attempt at renaming:- Copier since thats what it does. Let me know if others have better ideas